test/controllers/api/v2/ostree_branches_controller_test.rb

```diff
@@ -67,10 +67,21 @@ def test_compare
 
     def test_branch_order
```

Comment:
nitpick: we usually name controller test methods test_(controller method name)_(what we are testing)

so I think a better name would be test_index_ordering or something like that